templates/screen.service

```diff
@@ -5,7 +5,6 @@ After=init_display.service
 [Service]
 Type=simple
 ExecStart=${PROJECT_PATH}/venv/bin/python3 ${PROJECT_PATH}/src/screen/screen.py
```

Comment:
Quotes around `${PROJECT_PATH}`

Reply:
This is actually not standard, at least in systemd service files as the environment variable expansion is handled by systemd.